packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -184,6 +188,19 @@ export class SCWSigner implements Signer {
       // Sub Account Support
       case 'wallet_addSubAccount':
         return this.addSubAccount(request);
+      case 'coinbase_fetchPermissions': {
+        assertFetchPermissionsRequest(request);
+        const completeRequest = fillMissingParamsForFetchPermissions(request);
+        const permissions = (await fetchRPCRequest(
+          completeRequest,
+          CB_WALLET_RPC_URL
+        )) as FetchPermissionsResponse;
+        const requestedChainIdInHex = completeRequest.params?.[0].chainId;
```

Comment:
Will this always be a hex? The config is currently using number to be consistent with appChainIds?

Reply:
ya this will be always be hex, `assertFetchPermissionsRequest` will throw if app passed in non hex chain id, and `fillMissingParamsForFetchPermissions` will convert to hex for default's case

Reply:
Sounds good.